tests/nosetests/pyanaconda_tests/module_network_test.py

```diff
@@ -1158,3 +1160,16 @@ def apply_boot_options_test(self):
                 "ens11",
             ]
         )
+
+    def _is_using_persistent_device_names_test(self):
+        """Test _is_using_persistent_device_names."""
+        mocked_kernel_args = {'ifname': "bla bla"}
+        self.assertEqual(
+            self.network_module._is_using_persistent_device_names(mocked_kernel_args),
+            False
+        )
+        mocked_kernel_args = {'net.ifnames.prefix': "net"}
+        self.assertEqual(
+            self.network_module._is_using_persistent_device_names(mocked_kernel_args),
+            True
+        )
```

Comment:
I don't think this test is what should be here. Point is to test the use case, that means that we are trying to achieve something by the call of something. This test is testing only our protected function which is only a part in the whole solution and should be hidden as uninteresting part of the whole test. 
In general testing of private API is not what should be tested.

Reply:
Removed. After merging the branch I'll open a PR for update of installation task unit tests that should cover also the private method.